starcompactor/transforms/masker.py

```diff
@@ -55,11 +55,12 @@ def mask_fields(trace, trace_type, masker):
     return trace
 
 def ordered_mask(trace, field_name, ordered_list):
-    if trace[field_name]:
-        trace[field_name] = ordered_list.index(trace[field_name])
-        if trace[field_name] < 0: trace[field_name] = None
-    else:
-        trace[field_name] = None
+    if field_name in trace:
```

Comment:
Doesn't `index` raise an error if the item is not in the list?

```suggestion
    field_value = trace.setdefault(field_name, None)
    if field_value:
        try:
          trace[field_name] = ordered_list.index(field_value)
        except ValueError:
          pass
    return trace
```